.github/workflows/ci.yml

```diff
@@ -45,3 +45,25 @@ jobs:
 
       - name: Verify that canonical-data.json files adhere to canonical-schema.json
         run: yarn test
+
+      - name: Verify that reimplements-values refer to existing UUID
+        run: |
+          fail=0
+
+          for f in exercises/*/canonical-data.json; do
+              # Search for reimplemented test cases
+              reimplements=$(grep '"reimplements":' "$f" | sed -E 's/.*"([^"]+)".*/\1/')
```

Comment:
You could possibly use jq here too, see https://github.com/exercism/problem-specifications/pull/1700/files#r504241547.

Reply:
I don't fully understand it and if seems to be slower so I'd prefer the naive grep/sed approach

Reply:
The downside of the grep/sed approach is that it depends on formatting: if for any reason the reimplemented UUID drops down a line, sed won't see it anymore. jq is slower (because it's parsing the JSON), but it's immune to formatting (because it's parsing the JSON). I have never yet encountered a case where jq is sufficiently slower than sed that it makes a difference in human terms.

Reply:
Yeah, I assumed they were formatted correctly because that's something that (eventually) will be checked by CI as well.

Reply:
```suggestion
              reimplements=$(jq -r '.. |."reimplements"? | select(. != null)' "$f")
```